docs/technical/shell-integration.md

```diff
@@ -28,6 +28,9 @@ These shells are currently supported:
  - [PowerShell Core](https://github.com/powershell/powershell/)
  - [Hyper](https://hyper.sh/)
  - Git Bash (from [Git for Windows](https://git-for-windows.github.io/))
+ - [Cygwin](https://www.cygwin.com/)
+ - [WSL](https://docs.microsoft.com/en-us/windows/wsl/about)
+ - [Windows Terminal](https://github.com/microsoft/terminal)
```

Comment:
@AndreiMaga could we add a note that WSL is only on beta currently? thanks!

```suggestion
 - [Windows Terminal](https://github.com/microsoft/terminal) (on beta only)
```

Reply:
If I'm not mistaken, following the release of 2.4 this will be in "prod" the docs are not shipped so it seems to me like the beta only note is not necessary?

Reply:
@say25 i'm not aware of any discussion around flagging WSL detection for prod, but maybe i missed something? the feature flag is still beta-only:

https://github.com/desktop/desktop/blob/6c1800a06d2caaf9b6cebed8345cced1b32b4019/app/src/lib/feature-flag.ts#L84-L86

Reply:
@outofambit In my opition "only in beta" is not a good term for it as u can enable the functionality with an environment variable in "prod". Maybe a little "flag" like the one for dark mode would suit it better?
```suggestion
 - [WSL](https://docs.microsoft.com/en-us/windows/wsl/about) (beta)
```

Reply:
ohhh good point @AndreiMaga! that sounds good to me!

Reply:
ah i also just realized i suggested on the wrong line 🙈 sorry!

Reply:
that was the confusion Windows Terminal vs WSL 😅

Reply:
ohhhhh! sorry @say25 !